test/e2e-v2/cases/pulsar/expected/metrics-has-value-instance-label.yml

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type: TIME_SERIES_VALUES
+results:
+  {{- contains .results }}
+  - metric:
+      labels:
+        - key: _
+          value: *
+    values:
+      {{- contains .values }}
+      - id: {{ notEmpty .id }}
+        value: .
+        traceid: null
+      {{- end}}
+  {{- end}}
+error: null
```

Comment:
<img width="971" alt="image" src="https://github.com/apache/skywalking/assets/16773043/ae92100b-43e0-49e3-a267-3d11402e6935">
In fact, the e2e is failed. You used 1 expected file to verify all metrics and this expected file format is incorrect.
Please make sure no error in the e2e log.
you can refer the `vm's e2e`.

Reply:
This is due to the `value: *`.  Only the label part does not take effect.
```
type: TIME_SERIES_VALUES
results:
  {{- contains .results }}
  - metric:
      labels:
        - key: _
          value: *
    values:
      {{- contains .values }}
      - id: {{ notEmpty .id }}
        value: .
        traceid: null
      {{- end}}
  {{- end}}
error: null
```

Reply:
I will use two files to fix this.

Reply:
Fixed. There will be no error log.